homeassistant/components/dynalite/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "domain": "dynalite",
+    "name": "Direct Integration for Philips Dynalite",
+    "config_flow": true,
+    "documentation": "",
```

Comment:
Please add url to docs link on website.

Reply:
how do i know the docs link? Wouldn't it be merged at the same time this pull request goes into the code?

Reply:
this is pull request 10866 in the docs

Reply:
The docs link is `https://www.home-assistant.io/integrations/<domain>/`